conda/environments/env.yml

```diff
@@ -2,7 +2,7 @@ name: cucim
 channels:
   - conda-forge
 dependencies:
-  - cupy=9
+  - cupy>=9,<11.0.0a0
```

Comment:
Perhaps we should just drop this constraint? I think the original motivation was to ensure we got CuPy 9 as we needed the very latest features. Do things work without a constraint here?

Reply:
I am fine with just keeping the lower bound. I think it should tend to find 10.0 without the constraint, but it doesn't hurt to indicate that it is incompatible with 8.x.